packages/client/src/providers/ThemeTypeProvider/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { createContext, useState } from 'react';
+
+import useLocalStorage from '../../hooks/useLocalStorage';
+import { ThemeType } from '../../theme/Theme';
+import { ThemeProviderProps, ThemeTypeContextValue } from './ThemeTypeProvider.types';
+
+export const ThemeTypeContext = createContext<ThemeTypeContextValue>({
+  themeType: ThemeType.light,
+  setThemeType: () => null,
+});
+
+const ThemeTypeProvider = ({ themeType, children }: ThemeProviderProps) => {
+  const isSystemDarkTheme = window.matchMedia('(prefers-color-scheme: dark)');
+  const [localStorageType, setLocalStorageType] = useLocalStorage<ThemeType>('themeType');
+  const defaultType =
+    localStorageType || themeType || isSystemDarkTheme ? ThemeType.dark : ThemeType.light;
```

Comment:
add parentheses otherwise it will load dark instead of saved value in local storage

Reply:
it may be better to use the default value in local storage hook?